hack/lib

```diff
@@ -0,0 +1 @@
+../test/extended/testdata/cmd/hack/lib
```

Comment:
ok, I see

Reply:
> ok, I see

Pull out only the bits you need.  YOu cannot just copy things like rpm building.

Reply:
It feels more feasible to have just one symlink to the whole lib rather than linking specific deps.

I myself thought that the `hack/lib/build` would not be required but there are dependecies in `hack/lib/` and even other `hack/lib` scripts, also the `basicresources.sh` test depends on `hack/lib/build/version.sh`.

I could perhaps try to cut the dependencies inside the library itself to achieve some kind of isolation that probably should have been there from the start, not sure that it is worth cutting the symlink, though.